hack/run-canary.sh

```diff
@@ -17,10 +17,10 @@ do
   if [ $dir != "bitbucket.org" ]; then
     pushd "${FIXTURES}/$dir"
     rm -rf client models restapi cmd
-    swagger generate client --skip-validation
+    swagger generate client
```

Comment:
This change has change the CI's result on CircleCI docker build task:
skipping th validation alllowed to pass docker - now validation is enforced so we can't generate